app.rb

```diff
@@ -0,0 +1,168 @@
+class App
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def show_menu
+    puts "Welcome to School Library App!\n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```

Comment:
- [ ] Your methods for `Creating person`, `Creating book`, and `Creating rentals` are not working as expected, kindly fix this so your app is working properly

The output from the terminal:
![Screenshot_20220806_153010](https://user-images.githubusercontent.com/25227022/183253361-3e3b10c4-3708-4e65-a5c2-1208c4248015.png)
![Screenshot_20220806_153032](https://user-images.githubusercontent.com/25227022/183253362-500533b9-2b23-4c11-8031-b9abfb3b4af0.png)
![Screenshot_20220806_152929](https://user-images.githubusercontent.com/25227022/183253348-a1fdd737-693b-40fb-9659-02e18e54af36.png)